autoendpoint/src/settings.rs

```diff
@@ -46,13 +52,15 @@ impl Default for Settings {
             host: "127.0.0.1".to_string(),
             endpoint_url: "".to_string(),
             port: 8000,
+            db_dsn: None,
+            db_settings: "".to_owned(),
             router_table_name: "router".to_string(),
             message_table_name: "message".to_string(),
             /// max data is a bit hard to figure out, due to encryption. Using something
             /// like pywebpush, if you encode a block of 4096 bytes, you'll get a
             /// 4216 byte data block. Since we're going to be receiving this, we have to
-            /// presume base64 encoding, so we can bump things up to 5624 bytes max.
-            max_data_bytes: 5624,
+            /// presume base64 encoding, so we can bump things up to 5630 bytes max.
+            max_data_bytes: 5630,
```

Comment:
Why's this changing in this PR?

Reply:
For reference: bumping this because some envs seem to be exceeding the 5624 value somehow in the integration test's `test_big_message`.